src/features/classification/Classification.js

```diff
@@ -30,15 +35,18 @@ type Props = {
     name?: string,
     onClick?: (event: SyntheticEvent<HTMLButtonElement>) => void,
     shouldDisplayAppsAsIntegrations?: boolean,
+    shouldUseAppliedByLabels?: boolean,
 };
 
 const Classification = ({
+    aiClassificationReason,
     className = '',
     color,
     controls,
     controlsFormat,
     definition,
     isImportedClassification = false,
+    isLoadingAppliedBy = false,
```

Comment:
kept this as `isLoadingAppliedBy` generic because in theory in future, we can load this information outside of AI reasoning for Classification. But it is being used for AI reasoning